src/components/CodeAssignmentModal/emailTemplate.js

```diff
@@ -1,16 +1,16 @@
 const emailTemplate = {
-  greeting: `Your learning manager has provided you with a new access code to take a course at edX.
```

Comment:
Yay for consistency in capitalization! But should _Learning Manager_ be capitalized?

Reply:
A good question! I had pondered this as well. In the end, I read it as a formal title so I thought it should (would?) be capitalized. I'm not set on this if you think otherwise. I just wanted to try and make it consistent.